src/sqlfluff/rules/L006.py

```diff
@@ -64,7 +64,7 @@ def _missing_whitespace(seg, before=True):
         if seg.name.endswith("_bracket"):
             if seg.name.startswith("start_" if before else "end_"):
                 return False
-        if seg.is_meta:  # pragma: no cover
+        if seg.is_meta:
```

Comment:
This wasn’t one you added so let’s revert that.

```suggestion
        if seg.is_meta:   # pragma: no cover
```